package.json

```diff
@@ -57,7 +58,9 @@
     "@types/jest": "^24.0.12",
     "black-fmt": "https://github.com/learningequality/black-fmt#v0.1.3",
     "kolibri-tools": "0.13.0-dev.10",
-    "yarn-run-all": "^3.1.1"
+    "yarn-run-all": "^3.1.1",
+    "stylelint": "^13.3.3",
+    "stylelint-selector-tag-no-without-class": "^2.0.3"
```

Comment:
`stylelint` is already a dependency of `kolibri-tools` and does not need to be added here.

Additional `stylelint` related dependencies should also be added to the `Kolibri-tools` package.json file here: 

https://github.com/learningequality/kolibri/blob/4c961110eabf4711da4e2f6c1430c71004e1b6da/packages/kolibri-tools/package.json#L64-L71

Reply:
I've been read the yarn workspace document. I'll take note.